src/Components/Employees/EmployeeRow/employeeRow.js

```diff
@@ -3,9 +3,18 @@ import React from 'react';
 const EmployeeRow = ({ employee, setShowModal, setSelectedEmployee }) => {
   return (
     <tr className="rows">
-      <td> {employee._id} </td>
-      <td> {employee.name} </td>
-      <td> {employee.email} </td>
+      <td onClick={() => (window.location.href = `employees/form?id=${employee._id}`)}>
+        {employee.name}
+      </td>
+      <td onClick={() => (window.location.href = `employees/form?id=${employee._id}`)}>
+        {employee.lastName}
+      </td>
+      <td onClick={() => (window.location.href = `employees/form?id=${employee._id}`)}>
+        {employee.email}
+      </td>
+      <td onClick={() => (window.location.href = `employees/form?id=${employee._id}`)}>
+        {employee.phone}
+      </td>
       <td>
         <button
           onClick={() => {
```

Comment:
We have to stop the propagation here
```suggestion
          onClick={(event) => {
            event.stopPropagation(); // This is going to avoid the redirect to the form page when the user clicks on the button.
```